### DISCUSSION TITLE
Lack of specific metrics after yolov5 update

### DISCUSSION THREAD

**Discussion by AdGw** [ultralytics/yolov5#8705] [Q&A]
Hi!
I would like to ask about metrics. I started using YOLOv5 about January as well as Wandb tool for visualization. In default after every single epoch I was able to see evaluation of metrics/mAP_50 for every single class. Unfortunately after update few days ago and also update Pytorch stuff to newest version I realized that I have only few metrics related to train/val box loss, cls_loss, obj_loss without previous metrics which I mentioned above. Of course I can launch verbose flag but I'm wondering if it's possible to turn it on again in a simple manner without interfere in YOLOv5 library and see some visualizations as well as to compare previous runs from previous YOLOv5 version on the same graphs for every single class.

Example of what happens now:
![image](https://user-images.githubusercontent.com/26072039/180649090-148b3b16-f3b9-462c-9070-932b10109e2b.png)

**Comment by glenn-jocher** [comment]
@AdGw I think your code customizations are confusing the question. YOLOv5 has never reported per-class metrics to W&B. The default metrics which are reported are mAP@0.5, mAP@0.5:0.95, Precision and Recall.

**Reply by AdGw** [reply]
Ok I found the problem, it's quite tricky:

The problem was, that I installed yolov5 by "pip install yolov5" few months ago. I was confused why I lost the metrics but also why my training process is about 2x faster now than before.
Thanks for reply,
